web/src/beta/components/SceneSectionField/item.tsx

```diff
@@ -0,0 +1,72 @@
+import { useCallback } from "react";
+
+import Icon from "@reearth/beta/components/Icon";
+import Text from "@reearth/beta/components/Text";
+import { SwitchField } from "@reearth/beta/hooks/useManageSwitchState/hooks";
+import { styled, useTheme } from "@reearth/services/theme";
+
+export type SceneSectionItemProps = {
+  label: string;
+};
+export type Props = {
+  scene: SwitchField<SceneSectionItemProps>;
+  onActive?: (id: string) => void;
+  onAction?: (id: string) => void;
+};
+
+const SceneSectionItem: React.FC<Props> = ({ scene, onActive, onAction }) => {
+  const theme = useTheme();
+  const handleActive = useCallback(() => {
+    onActive?.(scene.id);
+  }, [scene.id, onActive]);
```

Comment:
Same as other PR. This needs `stopPropagation()`or else clicking the action button executes handleAction too